### ISSUE TITLE
Clean history (delete all messages) not working

### ISSUE THREAD

**Issue by dinhduongson** [RocketChat/Rocket.Chat.ReactNative#1472]
- Your Rocket.Chat Experimental app version: 4.1.0
- Your Rocket.Chat server version: 2.3.0
- Device (or Simulator) you're running with: Huawei Y9 2019 (Android 9.1.0), iPhone X Simulator

After I clean history of a room, using /api/v1/rooms.cleanHistory, messages still show.
They only disappear after I re-login.
Delete single message still work.

**Comment by diegolmello** [comment]
@dinhduongson It's not implemented yet.

**Comment by devyaniChoubey** [comment]
can i work  on this issue?

**Comment by Aaryan-kapur** [comment]
If this is available, may i take  this up? @diegolmello

**Comment by djorkaeffalexandre** [comment]
@Aaryan-kapur yes you can work on issues that don't have any Pull Requests opened yet or that have opened pull requests and stay without activity, you don't need to ask if you can work on some issue, just open a Pull Request and we'll review your changes.
Thanks.

**Comment by Aaryan-kapur** [comment]
Thank you, sir!

**Comment by guilhermedfs** [comment]
Is this bug still happening or this issue was closed?

**Comment by Typical-techno** [comment]
Is the issue still open ??

**Comment by RajvardhanT7747** [comment]
I am interested to work on this issue
